final-project-group-12

justgaret23 opened the pull request.

justgaret23 committed:
initial commit

justgaret23 committed:
gitignore

justgaret23 committed:
proposal

Commit:
added template code

charlieroberts commented:
This sounds like a pretty big project! How's it going so far? I'd encourage you do figure out a minimum-viable product / project and actively work towards that.

Do you have a sense of how the general UI will look? I'm particularly interested in the ability to—if I understand correctly—drill down into nested calendars while also viewing the calendar as a whole. Sounds like a great idea but I am having a bit of trouble picturing how it works.

Overall, great idea, but it sounds like a *lot* of work... please let me know if you develop a better sense of what your MVP would be for the project, and how you could potentially iterate from there.

gdgarsson committed:
New Controller Scripts

Added Database.js scripts for Controller, Event.js class file, empty scripts.js file for the front-end.

Commit:
React up and running

Commit:
Merge branch 'main' of https://github.com/justgaret23/final_project into main

justgaret23 committed:
add form

Commit:
Merge branch 'main' of https://github.com/justgaret23/final_project into main

justgaret23 committed:
chaos?

justgaret23 committed:
Merge branch 'main' of https://github.com/justgaret23/final_project into main

justgaret23 committed:
bootstrap

Commit:
changed to jsx

justgaret23 committed:
frontend stuff

gdgarsson committed:
Update Database.jsx

Updated Database.jsx to have prototype functions.

Commit:
mongodb integration done with schemas; all endpoints fully working

Commit:
github oauth done; changed files to make it work

Commit:
changed the getUser to send only id

Commit:
changed gitignore to have the build directory

Commit:
Delete build directory

Commit:
added endpoints for getting db data

Commit:
Merge branch 'main' of https://github.com/justgaret23/final_project

Commit:
changed index to be in line with other jsx files

Commit:
changed index.jsx filename to be more correct

Commit:
changed database file; rename and export functions

Commit:
made database file make more sense

Commit:
added a file for global variables

Commit:
changed database utils to work as intended

Commit:
changed databaseUtils to work as intended p2

Commit:
added loading to webpage for global vars

Commit:
added calendar sidebar items

Commit:
calendar sidebar working?

Commit:
calendar sidebar items working?

gdgarsson committed:
Create Task.jsx

gdgarsson committed:
Merge branch 'main' of https://github.com/justgaret23/final_project

gdgarsson committed:
Update Sidebar to include Tasks

Commit:
working on selecting calendar; not currently work

Commit:
Merge branch 'main'

Commit:
Task working as expected

Commit:
made sidebar thingy with task vr cool

Commit:
wtf another handle change

gdgarsson committed:
Update Sidebar.jsx

Commit:
trying to select calid

Commit:
Merge branch 'main' of https://github.com/justgaret23/final_project

Commit:
fixed some database utils

gdgarsson committed:
asdf

Commit:
changed some stuff

Commit:
Merge branch 'main' of https://github.com/justgaret23/final_project

gdgarsson committed:
Update Task.jsx

justgaret23 committed:
transferring work to desktop

Commit:
database updates fixed and cal children done

Commit:
Merge branch 'main' of https://github.com/justgaret23/final_project

Commit:
updated deleting functions to only delete one thin

Commit:
delete calendar ui done (with bug?)

gdgarsson committed:
asdf

gdgarsson committed:
zasdf

Commit:
wokring on modify calendar

Commit:
Merge branch 'main' of https://github.com/justgaret23/final_project

justgaret23 committed:
garrett's changes

justgaret23 committed:
hm today i will cause problems

justgaret23 committed:
die build you dont belong in this world

justgaret23 committed:
Merge pull request #1 from justgaret23/garrett

changes

Commit:
changed some stuff; working on tasks

Commit:
Merge branch 'main' of https://github.com/justgaret23/final_project

Commit:
everything works, goodnight

justgaret23 committed:
minor package.json mistake

Commit:
done?

Commit:
Merge branch 'main' of https://github.com/justgaret23/final_project

justgaret23 committed:
change start command

justgaret23 committed:
README

justgaret23 committed:
Final ReadMe

justgaret23 committed:
wording

justgaret23 committed:
link

justgaret23 committed:
format
